splinter/driver/webdriver/__init__.py

```diff
@@ -748,6 +748,9 @@ def __init__(self, element, parent):
     def _get_value(self):
         return self["value"] or self._element.text
 
+    def get(self,value):
```

Comment:
This is incorrect. The implementation of getting with a list index must be done through `__getitem__`

Reply:
To clarify:

I don't see a reason to add a `get` method.

However, if you were, you'd need to add it to the ElementAPI, which is the blueprint for WebDriverElement, and then support it across all the Elements, not just WebDriver.

Returning `self[value]` isn't what #777 was about. You would want to return the html attributes.

Reply:
Valid point, I'll work towards the review points.